docs/docs/css.md

```diff
@@ -3,7 +3,9 @@ title: Customizing Styles with CSS
 sidebar_label: Custom CSS
 ---
 
-You can add your own stylesheet in **Admin** > **Configure** > **Advanced** > **Custom CSS**.
+Coral uses [Shadow DOM Encapsulation](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM) to isolate the styling context from the parent page, therefore any CSS on the parent page will not apply to the embed.
+
+You can add your own stylesheet to customize Coral in **Admin** > **Configure** > **Advanced** > **Custom CSS**. Use a separate stylesheet to import custom fonts via @font-face, and add it in **Admin** > **Configure** > **Advanced** > **Custom Fonts CSS**.
```

Comment:
Since both of these are under **Admin** > **Configure** > **Advanced** > **Custom CSS**, maybe specify **Admin** > **Configure** > **Advanced** > **Custom CSS** > **Custom CSS Stylesheet URL** and **Admin** > **Configure** > **Advanced** > **Custom CSS** > **Custom CSS Stylesheet URL for Font Faces** for clarity.